### PR TITLE
Remove RuntimeEnvironment and target netcoreapp2.0

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

**Comment by rynowak** [comment]
Don't we still have lots of test projects that target netstandard? (Microsoft.Extensions, Razor, EF)

**Comment by BrennanConroy** [comment]
The only issue I found was https://github.com/aspnet/Razor/issues/1292

File: test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs [hunk 1]
```diff
@@ -32,7 +33,7 @@ public void RunTest_DoesNotRunOnWin7OrWin2008R2()
         {
             Assert.False(
                 RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
-                Microsoft.Extensions.Internal.RuntimeEnvironment.OperatingSystemVersion.StartsWith("6.1"),
+                Environment.OSVersion.Version.ToString().StartsWith("6.1"),
```
Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: Is this api in netstandard2.0?

Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: I believe so

Comment (anurse) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: https://apisof.net/catalog/System.Environment.OSVersion

Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: Nice try Andrew.

image

Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: @BrennanConroy can we use netstandard2.0 then? As this isn't a library we ship, I think we're not required to restrict this to netcoreapp2.0 only.

Comment (anurse) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: @blowdart would be proud. But it is a legit site. It got hit with the StartCom de-trust in Chrome. See https://github.com/terrajobst/apisof.net/issues/3

Comment (blowdart) [inline test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs]: This makes me happy

**Pull request merged**